core/sqf/sqenvcom.sh

```diff
@@ -29,13 +29,16 @@
 #  * Add an entry in sqf/LocalSettingsTemplate.sh to make path configurable.
 ##############################################################
 
+#Product version (Trafodion, Enterprise or Enterprise Adv)
```

Comment:
Instead of using the terms "Enterprise" or "Enterprise Adv", perhaps it would be better to say, "(Trafodion or derivative product)"